iot_config/src/org_service.rs

```diff
@@ -1,26 +1,39 @@
-use crate::{lora_field, org, GrpcResult, Settings, HELIUM_NET_ID};
+use crate::{
+    lora_field,
+    org::{self, OrgStatus},
+    route::list_routes,
+    GrpcResult, Settings, HELIUM_NET_ID,
+};
 use anyhow::Result;
 use file_store::traits::MsgVerify;
 use helium_crypto::{Network, PublicKey};
 use helium_proto::services::iot_config::{
-    self, OrgCreateHeliumReqV1, OrgCreateRoamerReqV1, OrgDisableReqV1, OrgDisableResV1,
+    self, ActionV1, OrgCreateHeliumReqV1, OrgCreateRoamerReqV1, OrgDisableReqV1, OrgDisableResV1,
     OrgEnableReqV1, OrgEnableResV1, OrgGetReqV1, OrgListReqV1, OrgListResV1, OrgResV1, OrgV1,
+    RouteStreamResV1,
 };
 use sqlx::{Pool, Postgres};
+use std::sync::Arc;
+use tokio::sync::broadcast::Sender;
 use tonic::{Request, Response, Status};
 
 pub struct OrgService {
     admin_pubkey: PublicKey,
     pool: Pool<Postgres>,
     required_network: Network,
+    route_update_tx: Arc<Sender<RouteStreamResV1>>,
```

Comment:
Why is the arc needed at all? a broadcast:Sender is Clone right?

Reply:
It is Clone, but I took this paragraph to mean it wasn’t safe to clone across threads: https://docs.rs/tokio/latest/tokio/sync/broadcast/fn.channel.html

Reply:
Since I need to share the sender with _from_ the router service where it’s created and the route stream lives

Reply:
it's safe to be clone'd within the same process and even across processes but then you need an Arc to pass it across a process boundary for lifetime reasons..  which means you really only need to have an Arc if the compiler complains that it can't figure out the lifetime. 

Did you try it without?

Reply:
nope, but it appears to work just fine; updated here [65c1533](https://github.com/helium/oracles/pull/313/commits/65c15331459a8788d93e5668ed359da8f74fa611)